src/conf.py

```diff
@@ -57,7 +57,8 @@
     'sphinxcontrib.jquery',
     'sphinx.ext.todo',
     'sphinx.ext.coverage',
-    'sphinx.ext.mathjax'
+    'sphinx.ext.mathjax',
+    'sphinxcontrib.jquery'
```

Comment:
This is already on line 57

Reply:
D'oh - I was too reliant on diff highlighting.

Reply:
interesting... I'll play around with this a bit.

Reply:
Looks to be only "develop" branch does not have the search working.

Reply:
Last time this happened, @kennyweiss helped by inspecting the html errors via the browser. This line specifically is what fixed it.

Reply:
Scratch that, works when I switch browsers/use a private tab. Probably just need to clear cache.